test_runner/fixtures/neon_fixtures.py

```diff
@@ -2237,6 +2239,13 @@ def get_test_output_dir(request: Any) -> pathlib.Path:
     return test_dir
 
 
+ATTACHMENT_SUFFIXES = frozenset((
+    '.log',
+    '.stderr',
+    '.stdout',
+))
```

Comment:
```suggestion
ATTACHMENT_SUFFIXES = frozenset((
    '.log',
    '.stderr',
    '.stdout',
    '.diffs',
))
```
The Postgres regression tests output `regression.diffs` on failure. It contains the diff between expected and actual results.